gluten-ut/spark34/src/test/scala/org/apache/spark/sql/sources/GlutenInsertSuite.scala

```diff
@@ -623,6 +624,24 @@ class GlutenInsertSuite
       }
     }
   }
+
+  testGluten("GLUTEN-7213: Check fallback reason with CheckOverflowInTableInsert") {
```

Comment:
Can we put the test into a common place? E.g.,
https://github.com/apache/incubator-gluten/tree/a7c9567bc7acdd14c8474c3d2dc99d1cfd062b56/gluten-ut/test/src/test/scala/org/apache/gluten/expressions

You can use `testWithSpecifiedSparkVersion` to let it be tested with spark-3.5.

Reply:
thanks, changed